ios/Runner/AppDelegate.swift

```diff
@@ -75,6 +80,40 @@ import Flutter
             }
         })
         
+        let unstoppableDomainChannel = FlutterMethodChannel(name: "com.cakewallet.cake_wallet/unstoppable-domain", binaryMessenger: controller.binaryMessenger)
+        unstoppableDomainChannel.setMethodCallHandler({ [weak self]
+                            (call: FlutterMethodCall, result: @escaping FlutterResult) -> Void in
+                            switch call.method {
+                            case "getUnstoppableDomainAddress":
+                                guard let args = call.arguments as? Dictionary<String, String>,
+                                      let domain = args["domain"],
+                                      let ticker = args["ticker"] else {
+                                    result(nil)
+                                    return
+                                }
+                                
+                                guard let resolution = self?.resolution else {
+                                    result(nil)
+                                    return
+                                }
```

Comment:
Combine all cases for `guard` condition in one `guard` instruction (move let resolution = self?.resolution into `guard` condition upper)